ledger/acctupdates.go

```diff
@@ -1219,9 +1225,11 @@ func (au *accountUpdates) getCreatorForRound(rnd basics.Round, cidx basics.Creat
 		}
 		// Check the database
 		creator, ok, dbRound, err = au.accountsq.lookupCreator(cidx, ctype)
-
+		if err != nil {
+			return basics.Address{}, false, err
+		}
 		if dbRound == currentDbRound {
-			return
+			return creator, ok, err
```

Comment:
I guess this is one more place I'd prefer to see `nil` than `err` so that it's very obvious this is the good path.

Reply:
thanks, done